browser/src/control/Control.PartsPreview.js

```diff
@@ -226,8 +226,20 @@ L.Control.PartsPreview = L.Control.extend({
 			}
 			if (app.file.fileBasedView)
 				this._map._docLayer._checkSelectedPart();
+			img.focus();
 		}, this);
 
+		var that = this;
+		img.onfocus = function () {
+			that.partsFocused = true;
+			that._map._clip.setTextSelectionType('text');
+		};
+
+		img.onblur = function () {
+			that.partsFocused = false;
+			that._map._clip.clearSelection();
+		};
+
```

Comment:
Do these onfocus/onblur changes mean that if you have something copied in the "main pane" and then click on the slide list that you lose the contents of the clipboard. Or what's the _clip.setTextSelection/clearSelection for ?